engine/installation/linux/oracle.md

```diff
@@ -1,204 +1,236 @@
-architecture. Docker requires the use of the Unbreakable Enterprise Kernel
-Release 4 (4.1.12) or higher on Oracle Linux. This kernel supports the Docker
-btrfs storage engine on both Oracle Linux 6 and 7.
+### OS requirements
 
-## Install
+To install Docker, you need the 64-bit version of Oracle Linux 6 or 7, running
+the Unbreakable Enterprise Kernel Release 4 (4.1.12) or higher. This kernel
+supports the Docker `btrfs` storage engine.
 
+### Remove unofficial Docker packages
 
-> **Note**: The procedure below installs binaries built by Docker. These binaries
-> are not covered by Oracle Linux support. To ensure Oracle Linux support, please
-> follow the installation instructions provided in the
-> [Oracle Linux documentation](https://docs.oracle.com/en/operating-systems/?tab=2).
->
-> The installation instructions for Oracle Linux 6 and 7 can be found in [Chapter 2 of
-> the Docker User&apos;s Guide](https://docs.oracle.com/cd/E52668_01/E75728/html/docker_install_upgrade.html)
+Oracle's repositories contain an older version of Docker, with the package name
+`docker` instead of `docker-engine`. If you installed this version of Docker,
+remove it using the following command:
 
+```bash
+$ sudo yum -y remove docker
+```
 
-1. Log into your machine as a user with `sudo` or `root` privileges.
-
-2. Make sure your existing yum packages are up-to-date.
-
-        $ sudo yum update
-
-3. Add the yum repo yourself.
-
-    For version 6:
-
-        $ sudo tee /etc/yum.repos.d/docker.repo <<-EOF
-        [dockerrepo]
-        name=Docker Repository
-        baseurl=https://yum.dockerproject.org/repo/main/oraclelinux/6
-        enabled=1
-        gpgcheck=1
-        gpgkey=https://yum.dockerproject.org/gpg
-        EOF
-
-    For version 7:
-
-        $ cat >/etc/yum.repos.d/docker.repo <<-EOF
-        [dockerrepo]
-        name=Docker Repository
-        baseurl=https://yum.dockerproject.org/repo/main/oraclelinux/7
-        enabled=1
-        gpgcheck=1
-        gpgkey=https://yum.dockerproject.org/gpg
-        EOF
-
-4. Install the Docker package.
-
-        $ sudo yum install docker-engine
+The contents of `/var/lib/docker` are not removed, so any images, containers,
+or volumes you created using the older version of Docker are preserved.
 
-5. Start the Docker daemon.
+## Install Docker
 
-     On Oracle Linux 6:
+You can install Docker in different ways, depending on your needs:
+
+- Most users
+  [set up the official Docker repositories](#install-using-the-repository) and
+  install from them, for ease of installation and upgrade tasks. This is the
+  recommended approach.
+
+- Some users download the RPM package and install it manually and manage
+  upgrades completely manually.
+
+- Some users cannot use third-party repositories, and must rely on
+  the version of Docker in the Red Hat repositories. This version of Docker may
+  be out of date. Those users should consult the Red Hat documentation and not
```

Comment:
A link to the Oracle documentation that outlines the installation and upgrade procedures should be included. It was included in the previous version of the Docker documentation.